xbmc/platform/darwin/osx/SDLMain.mm

```diff
@@ -556,7 +556,8 @@ int main(int argc, char *argv[])
     status = SDL_main(gArgc, gArgv);
     SDL_Quit();
 
-    [xbmc_delegate applicationWillTerminate:NULL];
```

Comment:
this is an old warning and can be ignored in this PR

Reply:
So then the question will be where is the 6th new warning as I only found 5!

Reply:
Although this makes 6 so Jenkins is happy now.

Reply:
I think is the last one: https://jenkins.kodi.tv/job/OSX-64/16848/warnings2Result/


ThreadImpl.cpp:65, kodi compiler warnings, Priority: Normal

'syscall' is deprecated: first deprecated in macOS 10.12 - syscall(2) is unsupported; please switch to a supported interface. For SYS_kdebug_trace use kdebug_signpost(). [-Wdeprecated-declarations]